.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please add `.vscode` to the `.gitignore` since this is an IDE specific file and shouldn't be required for everyone